modules/web-console/frontend/app/components/ui-grid/controller.js

```diff
@@ -75,6 +78,7 @@ export default class IgniteUiGrid {
         }
 
         this.grid = {
+            appScopeProvider: this.$scope.$parent,
```

Comment:
@Klaster1 Is it really a good idea to use scope here? Possibly communication between ui-grid and wrapping component could be better organized through services, not through $scope.

Reply:
How do you imagine a service would help here? appScopeProvider is a default ui-grid API designed to help with "transclusion" of cell templates. Because this particular grid wraps ui-grid in another isolated scope, appScopeProvider needs a bit of help to work as intended.